src/DomainHost.sol

```diff
@@ -306,17 +306,45 @@ abstract contract DomainHost {
         emit Deficit(wad);
     }
 
+    /// @notice Governance is accruing surplus
+    /// @param debt The amount of debt in the guest domain that should be backed by pre-minted DAI [WAD]
+    function accrue(uint256 debt) external auth returns (uint256 _wad) {
+        require(vat.live() == 1, "DomainHost/vat-not-live");
+        uint256 _dsin = dsin;
+        uint256 _ddai = ddai;
+        require(_ddai > _dsin, "DomainHost/no-surplus");
+
+        unchecked { _wad = _ddai - _dsin; }
+        dai.transferFrom(address(escrow), address(this), _wad);
+        daiJoin.join(address(vow), _wad);
+
+        int256 diff = _int256(debt) - _int256(grain);
+        if (diff > 0) {
+            vat.slip(ilk, address(this), diff);
+            vat.frob(ilk, address(this), address(this), address(this), diff, diff);
```

Comment:
This will fail if there isn't enough DC. Maybe that's fine.

Reply:
Yeah this is a good point, but as it is a governance action, they can also increase the debt ceiling at the moment executing it if necessary.

Reply:
Yeah it's `auth`ed so that's ok.